src/pages/MyPage.tsx

```diff
@@ -43,15 +39,22 @@ export default function MyPage() {
     { id: 1, title: '바닷가', boardList: boardList, savedTime: '방금' },
     { id: 2, title: '제주도', boardList: boardList, savedTime: '방금' },
   ];
-  const [userInfo, setUserInfo] = useState<MyPageUserInfo>();
+  const [userInfo, setUserInfo] =
+    useState<Pick<UserInfo, 'userId' | 'nickname' | 'followingCnt' | 'followerCnt'>>();
```

Comment:
```suggestion
    useState<Omit<UserInfo, 'id'>>();
```

Omit을 쓰면 더 깔끔할 것 같습니다!

Reply:
유틸리티 타입 정말 유용하네요😯👍 좋은 방법인 것 같습니다 감사해요 ㅎㅅㅎ